packages/nitro-protocol/src/contract/outcome.ts

```diff
@@ -134,7 +132,7 @@ export function encodeAssetOutcome(allocationOrGuarantee: Allocation | Guarantee
 export type AssetOutcome = AllocationAssetOutcome | GuaranteeAssetOutcome;
 export type Outcome = AssetOutcome[];
 // ^ Note this is not the same structure
-// as the Outcome struct defined in Outcome.sol
+// As the Outcome struct defined in Outcome.sol
```

Comment:
I'm not sure if we really need a rule to capitalize the start of comments, especially since many of comments are broken over multiple lines, so the capitalization doesn't make sense (like here).

Reply:
@lalexgap I think that we should, in this case, have a multiline comment instead of multiple single line ones?